common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -381,8 +381,17 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
     new RpcTimeout(get(RPC_LOOKUP_TIMEOUT).milli, RPC_LOOKUP_TIMEOUT.key)
   def rpcAskTimeout: RpcTimeout =
     new RpcTimeout(get(RPC_ASK_TIMEOUT).milli, RPC_ASK_TIMEOUT.key)
-  def rpcDispatcherNumThreads(availableCores: Int): Int =
-    get(RPC_DISPATCHER_THREADS).getOrElse(availableCores)
+  def rpcDispatcherNumThreads(availableCores: Int): Int = {
+    val num = get(RPC_DISPATCHER_THREADS)
```

Comment:
Could this method invoke `rpcDispatcherNumThreads(availableCores: Int, role: String)`?

Reply:
I think it can’t. It is a fallback condition in this pr.